pyGPG/gpg.py

```diff
@@ -159,6 +220,18 @@ def decrypt(self, inputtxt=None, inputfile=None, outputfile=None):
         '''
         return self.runGPG('decrypt', inputtxt, inputfile, outputfile)
 
+<<<<<<< HEAD
+=======
+    def encrypt(self, inputtxt, recipient):
+        '''encrypts the inputtxt block passed in
+        and/or returns the decrypted as GPGResult.output
+
+        @param inputtxt: string (optional)  of text to send to gpg's stdin
+        @rtype GnuPGResult object
+        '''
+        return self.runGPG2('encrypt', inputtxt, '--recipient',recipient, '--trust-mode', 'always', '--armor')
+
```

Comment:
Here is an example of how to use pyGPG's flexibility of being config driven.

https://github.com/gentoo/gentoo-keys/blob/master/gkeys/gkeys/lib.py#L246

If you made this encrypt() use that method of setting the task option, then called runGPG(...), then reset that task option to the default after the call (removes potential private info from memory).  That would be what I would like to add to pyGPG.